wordpress/composer.json

```diff
@@ -47,19 +57,12 @@
         "wpackagist-plugin/wp-bootstrap-blocks",
         "humanmade/s3-uploads",
         "wpackagist-plugin/login-lockdown",
-        "wpackagist-plugin/disable-user-login"
+        "wpackagist-plugin/disable-user-login",
+        "cds-snc/cds-base"
       ]
     }
   },
   "scripts": {
-    "post-install-cmd": [
-      "cd wp-content/themes/cds-default && composer install",
-      "cd wp-content/mu-plugins/cds-base && composer install"
-    ],
-    "post-update-cmd": [
-      "cd wp-content/themes/cds-default && composer update",
-      "cd wp-content/mu-plugins/cds-base && composer update"
-    ],
```

Comment:
Don't need this anymore as all dependencies will be installed in `wordpress/vendor`